ElementX/Sources/Screens/ComposerToolbar/View/VoiceMessageRecordingButton.swift

```diff
@@ -17,79 +17,60 @@
 import Compound
 import SwiftUI
 
+enum VoiceMessageRecordingButtonMode {
+    case standard
```

Comment:
```suggestion
    case idle
```
Maybe it describe better the button's state?
wdyt?

Reply:
👍